hack/test-stirunimage.sh

```diff
@@ -8,14 +8,14 @@ export PATH="$PWD/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH):$PAT
 
 function time_now()
 {
-  date +%s000
+    date +%s000
```

Comment:
Why are there so many whitespaces changes in the same commit?

It is considered good practice, and helps understanding the project history if changes in a commit are related to the same fix/feature/etc. This commit will be always explained as "SElinux ...".